src/DynamoCoreWpf/Views/Core/DynamoView.xaml.cs

```diff
@@ -1305,6 +1305,13 @@ private void Selection_CollectionChanged(object sender, NotifyCollectionChangedE
 
         private void Controller_RequestsCrashPrompt(object sender, CrashPromptArgs args)
         {
+            if (CrashReportTool.ShowCrashErrorReportWindow(
+                (args is CrashErrorReportArgs cerArgs) ? cerArgs : 
```

Comment:
what condition is this handling?

Reply:
`Controller_RequestsCrashPrompt` can be called with `CrashPromptArgs`(existing API) and with `CrashErrorReportArgs` (new API added). For both cases we want to call into CER first (if CER is not found, then we go with the backup crash prompt)
`CrashErrorReportArgs` is derived from `CrashErrorReportArgs` and has a bunch of flags to customize CER.
If `Controller_RequestsCrashPrompt` is called with `CrashErrorReportArgs`, then we pass it to CER.
If `Controller_RequestsCrashPrompt` is called with `CrashPromptArgs`, then we call CER with a new `CrashErrorReportArgs` (defaults to send everything it can to CER), basically ignoring the `CrashPromptArgs` args.

I can add some of this info to the APIs